public/js/submit_delete_request_data.js

```diff
@@ -92,8 +97,8 @@ $(document).ready(function() {
     $("#request-data").on("click", function(e) {
         // e.preventDefault();
 
-        let minDate = $("#start-date-download").val();
```

Comment:
why is this deleted?

Reply:
Again, I removed all my changes before I pulled and pushed... sooooo, not sure. Because it should have reverted back to where I didn't touch it.

Reply:
fixed